Assets/Scripts/Ball/Wind.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Wind : MonoBehaviour
+{
+    [SerializeField] float _windForce = 2f;
+
+    private void OnTriggerStay(Collider other)
+    {
+        var hitObj = other.gameObject;
+        if (hitObj != null)
+        {
+            var rb = hitObj.GetComponent<Rigidbody>();
+            var dir = transform.forward;
```

Comment:
You will want to promote this to a `[SerializedField]` field at the top so we can control it in the inspector. Seems like I made a mistake in direction, so having this be in the inspector would have made it easier to fix.